frontend/app/screens/SplatRouter/screens/BasicPage/index.js

```diff
@@ -35,6 +35,9 @@ const BasicPage = ({ title, body }: BasicPageProps): React.Element<any> => {
     case 'Program':
       backgroundImage = 'camper';
       break;
+    case 'Getting Around':
+      // falls through
+
```

Comment:
@WidgetsBurritos If you are going to format this with extra lines let's just add:
```
      backgroundImage = 'alamo';
      break;
```